src/Matrix/EigenVectors.cpp

```diff
@@ -62,9 +62,9 @@ EigenVectors& EigenVectors::operator=(const EigenVectors& r)
     _ready        = r._ready;
     _eigenValues  = r._eigenValues;
     _eigenVectors = r._eigenVectors;
-    _mat          = r._mat;
-    _nrows        = r._nrows;
-    _ncols        = r._ncols;
+    // _mat          = r._mat; // _mat is a reference, cannot be assigned
```

Comment:
I'm not sure if this `operator=` works correctly when the copy here is disabled.
I would remove altogether the whole `operator=` definition and let the compiler handle that itself (it should `delete` the method). Same for the copy constructor (rule of zero).